packages/auth-nextjs/src/shared.ts

```diff
@@ -93,6 +93,10 @@ export interface CreateAuthRouteHandlers {
   onSignout(req: NextRequest): Promise<never>;
 }
 
+type ReadonlyRequestCookies = Awaited<ReturnType<typeof nextCookies>>;
+const cookies: () => Promise<ReadonlyRequestCookies> | ReadonlyRequestCookies =
+  nextCookies;
```

Comment:
unnecessary, so can remove it, but just wanted to show that the return type depends on the next.js version used by the user

Reply:
Feel free to remove it in this PR, thanks for the demonstration!